libs/sdk-backend-spi/src/workspace/genAI/index.ts

```diff
@@ -0,0 +1,51 @@
+// (C) 2023-2024 GoodData Corporation
+
+import { GenAISemanticSearchType, ISemanticSearchResultItem } from "@gooddata/sdk-model";
+
+/**
+ * GenAI-powered semantic search service.
+ * @alpha
+ */
+export interface IGenAIService {
+    getSemanticSearchQuery(): ISemanticSearchQuery;
```

Comment:
Pls can you add comments ? for other fields too

Reply:
I'll include it in the next PR